curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b05367b59300bcb5f18ef.md

```diff
@@ -0,0 +1,36 @@
+---
+id: 657b05367b59300bcb5f18ef
+title: Task 1
+challengeType: 22
+dashedName: task-1
+---
+
+# --description--
+
+In English, you can use the verb `to be` to ask questions about feelings, states, or conditions. 
+When you want to ask such questions, you just swap the positions of the subject (like `you` or `he`) and the verb `to be` (like `are` or `is`). For example, to turn this sentence `You are a developer.` into a question, you switch `you` and `are`:
+
+`Are you a developer?` Notice how the order changes to ask a question.
+
+Here are some more examples of turning statements into questions using `to be`: 
+
+`He is here.` -> `Is he here?`
+`They are ready.` -> `Are they ready?`
```

Comment:
The formatting will need updating later.

Reply:
How so?